L/libgsd/build_tarballs.jl

```diff
@@ -14,6 +14,8 @@ sources = [
 script = raw"""
 cd ${WORKSPACE}/srcdir/gsd/
 mkdir -p "${libdir}"
+mkdir -p ${prefix}/share
+cp ./gsd/gsd.h ${prefix}/share/gsd.h
```

Comment:
```suggestion
install -D -m644 ./gsd/gsd.h ${includedir}/gsd.h
```
Header files should be in the include directory, not share.

(and using `install -D` will automatically create the path as required).